src/In.ProjectEKA.HipService/User/UserController.cs

```diff
@@ -48,11 +48,29 @@
~        }
~
~        [NonAction]
~        public async Task OnAuthConfirmFor(JObject response)
         {
             var authOnConfirmResponse = response.ToObject<AuthOnConfirmResponse>();
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=6173534)